src/components/Img/__mocks__/img.mocks.js

```diff
@@ -0,0 +1,9 @@
+export const mockImg = {
+  URLs: {
+    large: "https://picsum.photos/350/350?image=0",
+    medium: "https://picsum.photos/300/300?image=0",
+    original: "https://picsum.photos/1500/1500?image=0",
+    small: "https://picsum.photos/550/550?image=0",
+    thumbnail: "https://picsum.photos/50/50?image=0"
```

Comment:
Should we create some sort of repo / site to use for test images, that we know will always be live (or at least will be out fault if it's not live? Not sure if we should be using a third party to provide testing data like this.

Reply:
I'm just going to remove these, I only used them for writing the Img component styles.